### DISCUSSION TITLE
Window separator color

### DISCUSSION THREAD

**Discussion by mizlan** [kovidgoyal/kitty#5491] [Q&A] [answered]
My kitty has defaulted to a neon green window separator color. Is this possible to configure? See the border surrounding the bottom right window below:

<img width="1171" alt="Screen Shot 2022-09-07 at 8 57 58 PM" src="https://user-images.githubusercontent.com/44309097/189031069-aed98fb1-4187-45ee-a571-1dcc359074f2.png">

**Comment by kovidgoyal** [comment, marked as answer]
https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.active_border_color
